### ISSUE TITLE
Can't use decorator with `object | null` property.

### ISSUE THREAD

**Issue by RichiCoder1** [Microsoft/TypeScript#15181]
**TypeScript Version:**  2.2.2

**Code**

```ts
import { bindable, bindingMode } from "aurelia-framework";

export class MyThingCustomElement {
  @bindable({ defaultBindingMode: bindingMode.oneWay })
  public options: object | null = null;
}

```

**Expected behavior:**
Works.
**Actual behavior:**
```
      C:\Source\FA\Project\Frontend\node_modules\typescript\lib\typescript.js:64363
                      throw e;
                      ^
      Error: Debug Failure. False expression: Unexpected node.
      Verbose Debug Information: Node 133 (ObjectKeyword) was unexpected.
          at Object.assert (c:\Source\FA\ScriptPad-Frontend\ScriptPad.Frontend\node_modules\typescript\lib\typescript.js:3259:23)
          at Object.Debug.failBadSyntaxKind (c:\Source\FA\ScriptPad-Frontend\ScriptPad.Frontend\node_modules\typescript\lib\typescript.js:46001:55)
          at serializeTypeNode (c:\Source\FA\ScriptPad-Frontend\ScriptPad.Frontend\node_modules\typescript\lib\typescript.js:47932:30)
          at serializeUnionOrIntersectionType (c:\Source\FA\ScriptPad-Frontend\ScriptPad.Frontend\node_modules\typescript\lib\typescript.js:47941:44)
          at serializeTypeNode (c:\Source\FA\ScriptPad-Frontend\ScriptPad.Frontend\node_modules\typescript\lib\typescript.js:47922:28)
          at serializeTypeOfNode (c:\Source\FA\ScriptPad-Frontend\ScriptPad.Frontend\node_modules\typescript\lib\typescript.js:47793:28)
          at addOldTypeMetadata (c:\Source\FA\ScriptPad-Frontend\ScriptPad.Frontend\node_modules\typescript\lib\typescript.js:47713:92)
          at addTypeMetadata (c:\Source\FA\ScriptPad-Frontend\ScriptPad.Frontend\node_modules\typescript\lib\typescript.js:47707:17)
          at transformAllDecoratorsOfDeclaration (c:\Source\FA\ScriptPad-Frontend\ScriptPad.Frontend\node_modules\typescript\lib\typescript.js:47546:13)
          at generateClassElementDecorationExpression (c:\Source\FA\ScriptPad-Frontend\ScriptPad.Frontend\node_modules\typescript\lib\typescript.js:47593:40)
```
Was made slightly more annoying to pin down considering it only happened when I was running under the Webpack Middleware provided by aspnet/SpaServices

**Comment by aluanhaddad** [comment]
Can you verify the TypeScript version in use by ts-loader by running
```
> c:\Source\FA\ScriptPad-Frontend\ScriptPad.Frontend\node_modules\.bin\tsc --version
```
Microsoft.AspNetCore.SpaServices also appears to use a _very_ old version of ts-loader.

**Comment by RichiCoder1** [comment]
The console claimed to be using 2.2.2, and I believe it's just using my webpack configuration and pulling typescript from my local node_modules (which is also 2.2.2).

**Comment by sandersn** [comment]
Looking at `serializeTypeNode` in `master`, it does indeed fail to handle `ObjectKeyword`. I was able to repro this in a standalone test too. It's a simple oversight so it should be easy to fix.

**Comment by sandersn** [comment]
Fix is up at #15194.
